lib/inline_oneboxer.rb

```diff
@@ -74,4 +95,12 @@ def self.cache_key(url)
     "inline_onebox:#{url}"
   end
 
+  def self.post_author_for_title(topic, post_number)
+    guardian = Guardian.new
```

Comment:
this is a bit tricky, at least it is 100% safe though. 

the tricky thing is that in certain cases the author can be safely exposed (eg: you are in a PM and reference a post further up that is not a whisper) ... but this gets so complex to get right cause rouge editors can edit in a link in the case of secure categories and so on.... so it is probably best just to keep stuff simple and safe like you did.